### PR TITLE
Bug 1119312: Upgrade Django to 1.6.9

### PR DESCRIPTION
**Pull request by pmclanahan**
Also fixes some tests that were failing after upgrade.
Code was relying on Django's strip_tags filter which has
changed and now ignores things that obviously aren't HTML tags

Updated to latest playdoh-lib which includes fixes for Django 1.6
in funfactory.

@jgmize r?

### CODE REVIEW

File: bedrock/legal/templates/legal/emails/fraud-report.txt [hunk 1]
```diff
@@ -10,13 +10,13 @@ A new violating website report has been submitted with the following information
 {{ input_product }}
 
 + Specific product
-{{ input_specific_product|safe|strip_tags }}
+{{ input_specific_product|strip_tags|safe }}
```
Comment (jgmize) [inline bedrock/legal/templates/legal/emails/fraud-report.txt]: We should not do this: https://docs.djangoproject.com/en/1.7/ref/templates/builtins/#striptags

**Comment by jgmize** [comment]
Closing this in favor of #2635

File: bedrock/legal/templates/legal/emails/fraud-report.txt (hunk 1 above)
Comment (pmclanahan) [inline bedrock/legal/templates/legal/emails/fraud-report.txt]: Why not? This is a text email? It's what we've been doing and seems fine.

Comment (jgmize) [inline bedrock/legal/templates/legal/emails/fraud-report.txt]: It's *probably* fine in a plain text email , but there have been issues in the past where certain combinations of email client or web mail clients would ignore MIME types. That said, the recipients of these particular emails are highly unlikely to be using ancient versions of outlook express (at least I hope so!), so my knee-jerk reaction was probably a little strong given the context.

**Pull request closed without merging**